Project/package.json

```diff
@@ -32,11 +32,14 @@
     "@angular/platform-browser": "2.2.1",
```

Comment:
You need to add the video player plugin to package.json (under plugins), as well.